pkg/deploy/install.go

```diff
@@ -29,6 +29,10 @@ func (deploy *Deployer) installNamespace() error {
 		if deploy.config.Platform == "openshift" {
 			labels["openshift.io/cluster-monitoring"] = "true"
 			deploy.logger.Infof("Labeling the %s namespace with 'openshift.io/cluster-monitoring=true'", deploy.config.Namespace)
+			namespaceObjectMeta.Annotations = map[string]string{
```

Comment:
I think a comment along the lines of the commit message would be helpful to explain what's going on.

Reply:
Updated in the last force-push. I tried to add some context to both of the namespace metadata modifications we make.

Reply:
![image](https://cloud.githubusercontent.com/assets/4449199/19351172/ae881904-9128-11e6-9226-b3eef5ef90d2.png)